### ISSUE TITLE
Question about maxbins’s behavior 

### ISSUE THREAD

**Issue by kanitw** [vega/vega#469]
This is more like a datalib issue but since my debug scenario happens at the Vega-level, I’m reporting it here.  

For the following histogram, I get 4 bins when `maxbins` is `10`. 
However, if I change `maxbins` to `11`, I get 10 bins. 

I wonder if this behavior is correct?   

```
{
  "width": 200,
  "height": 200,
  "padding": "auto",
  "data": [
    {
      "name": "source",
      "url": "data/cars.json",
      "format": {
        "type": "json",
        "parse": {"Miles_per_Gallon": "number"}
      },
      "transform": [
        {
          "type": "filter",
          "test": "datum.Miles_per_Gallon!==null"
        },
        {
          "type": "bin",
          "field": "Miles_per_Gallon",
          "output": {
            "start": "bin_Miles_per_Gallon_start",
            "mid": "bin_Miles_per_Gallon_mid",
            "end": "bin_Miles_per_Gallon_end"
          },
          "maxbins": 10
        }
      ]
    },
    {
      "name": "summary",
      "source": "source",
      "transform": [
        {
          "type": "aggregate",
          "groupby": [
            "bin_Miles_per_Gallon_start",
            "bin_Miles_per_Gallon_mid",
            "bin_Miles_per_Gallon_end"
          ],
          "summarize": {"*": ["count"]}
        }
      ]
    }
  ],
  "marks": [
    {
      "name": "root",
      "type": "group",
      "from": {"data": "layout"},
      "properties": {
        "update": {"width": {"value": 200},"height": {"value": 200}}
      },
      "marks": [
        {
          "type": "rect",
          "properties": {
            "update": {
              "x": {
                "scale": "x",
                "field": "bin_Miles_per_Gallon_start",
                "offset": 1
              },
              "x2": {
                "scale": "x",
                "field": "bin_Miles_per_Gallon_end"
              },
              "y": {"scale": "y","field": "count"},
              "y2": {"field": {"group": "height"}},
              "fill": {"value": "#4682b4"}
            }
          },
          "from": {"data": "summary"}
        }
      ],
      "scales": [
        {
          "name": "x",
          "type": "linear",
          "domain": {
            "data": "summary",
            "field": [
              "bin_Miles_per_Gallon_start",
              "bin_Miles_per_Gallon_end"
            ]
          },
          "rangeMin": 0,
          "rangeMax": 200,
          "round": true,
          "clamp": true,
          "nice": true,
          "zero": false
        },
        {
          "name": "y",
          "type": "linear",
          "domain": {"data": "summary","field": "count","sort": true},
          "rangeMin": 200,
          "rangeMax": 0,
          "round": true,
          "clamp": true,
          "nice": true
        }
      ],
      "axes": [
        {
          "type": "x",
          "scale": "x",
          "format": "s",
          "grid": false,
          "title": "BIN(Miles_per_Gallon)",
          "properties": {
            "labels": {
              "angle": {"value": 270},
              "align": {"value": "right"},
              "baseline": {"value": "middle"}
            }
          }
        },
        {
          "type": "y",
          "scale": "y",
          "format": "s",
          "grid": true,
          "layer": "back",
          "title": "Number of Records"
        }
      ]
    }
  ]
}
```

**Comment by jheer** [comment]
Already fixed in datalib 1.5.7 :)

**Comment by kanitw** [comment]
Oh true!   I just rebuild Vega locally with datalib `1.5.7` and this works!  Thank you! :)
